localisation/000_wc_traits.csv

```diff
@@ -622,3 +622,7 @@ proud_desc;For some reason this character is proud or even selfish. Proud charac
 humble_desc;This character is humble and [From.GetHerHis] trait is expressed in moderation in [From.GetHerHis] demands, indifference to wealth and luxury, as well as [From.GetHerHis] determination to show [From.GetHerHis]self. Humble people tend to dislike proud ones.;;Dieser Charakter ist bescheiden und [From.GetHerHis] Charakterzug dr�ckt sich in [From.GetHerHis]en Forderungen aus, in Gleichg�ltigkeit gegen�ber Reichtum und Luxus sowie in [From.GetHerHis]er Entschlossenheit, sich selbst zu zeigen. Bescheidene Leute neigen dazu, Stolze nicht zu m�gen.;;;;;;;;;;;x
 pneumonic_desc;This character suffers from pneumonia � a deadly infection of the lungs.;;Dieser Charakter leidet an einer Lungenentz�ndung, einer oftmals t�dlichen Infektion.;;;;;;;;;;;x
 lefthanded_desc;This character is left-handed. People who are left-handed tend to have a considerable advantage in combat.;;Dieser Charakter ist Linksh�nder. Personen, die Linksh�nder sind, haben im Kampf oft einen erheblichen Vorteil.;;;;;;;;;;;x
+creature_wickerman;Wickerman;;;;;;;;;;;;;
+creature_wickerman_desc;This character is a wickerman. The wickermen are created by the Throsic covens from dead objects such as wood, flesh, fur and bones and animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust which are thus given physical form.;;;;;;;;;;;;;
```

Comment:
Having "the" in front of Throsic covens imply they all do it together when I feel they do it independently of one another. Feel free to correct it.
```suggestion
creature_wickerman_desc;This character is a wickerman. Wickermen are created by Throsic covens from inanimate organic materials such as wood, flesh, fur, and bones and are animated by ancient Drust death magic. They are powered by the souls of mortal victims or by the spirits of ancient Drust and thus give the spirits physical form.;;;;;;;;;;;;;
```

Reply:
What are Throsic covens even? Does throsic have to be with capital? Can't find the term on google, so I dunno...

Reply:
Throsic, as in related to the realm of Thros, so capitalization is correct.